builder.md

```diff
@@ -82,41 +87,68 @@ Each template repository MUST have a templates folder containing a config.yaml a
+- `output-directory-name/base16-default-dark.file-extension`, built from `default.mustache`.
+- `output-directory-name/base16-default-dark.another-extension`, built from `additional.mustache`.
 
 </details>
 
 ## Template Variables
 
 A builder MUST provide the following variables to template files:
 
-- `scheme-name` - obtained from the `scheme` key of the scheme file
-- `scheme-author` - obtained from the `author` key of the scheme file
-- `scheme-description` - obtained from the `description` key of the scheme file (fallback value: `scheme-name`)
-- `scheme-variant` - obtained from the `variant` key of the scheme file.
-- `is_{{variant}}_variant` - dynamic value obtained from the `variant` key of the scheme file. e.g. `variant: "light"` provides `is_light_variant` with a value of `true`. (fallback value: `is_dark_variant` with a value of `true`). Officially supported values are `light` and `dark`, but any string could theoretically be used.
-- `scheme-slug` - the scheme filename made lowercase, not including the `.yaml` extension
-- `base00-hex` to `base0F-hex` - 6-digit hex color value obtained from the scheme file. MUST NOT include a leading `#`. e.g "7cafc2".
-- `base00-hex-bgr` to `base0F-hex-bgr` - built from a reversed version of all the hex values e.g "c2af7c"
-- `base00-hex-r` to `base0F-hex-r` - red component of the hex color value. e.g "7c"
-- `base00-hex-g` to `base0F-hex-g` - green component of the hex color value. e.g "af"
-- `base00-hex-b` to `base0F-hex-b` - blue component of the hex color value. e.g "c2"
-- `base00-rgb-r` to `base0F-rgb-r` - red component as a value between `0` and `255`. e.g "124"
-- `base00-rgb-g` to `base0F-rgb-g` - green component as a value between `0` and `255`. e.g "175"
-- `base00-rgb-b` to `base0F-rgb-b` - blue component as a value between `0` and `255` e.g "194"
-- `base00-dec-r` to `base0F-dec-r` - red component as a value between `0` and `1.0`. e.g "0.4863"
-- `base00-dec-g` to `base0F-dec-g` - green component as a value between `0` and `1.0`. e.g "0.6863"
-- `base00-dec-b` to `base0F-dec-b` - blue component as a value between `0` and `1.0`. e.g "0.7608"
+- `scheme-name` - obtained from the `name` key of the scheme input
+- `scheme-author` - obtained from the `author` key of the scheme input
+- `scheme-description` - obtained from the `description` key of the scheme input
+- `scheme-slug` - obtained from the `slug` key of the scheme input (fallback value: a [slugified](#slugify) `scheme-name`)
+- `scheme-system` - obtained from the `system` key of the scheme input
+- `scheme-variant` - obtained from the `variant` key of the scheme input
+- `is_{{variant}}_variant` - dynamic value built from the `variant` key of the scheme file. e.g. `variant: "light"` provides `is_light_variant` with a value of `true`.
```

Comment:
So invalid variables are always just `false` and don’t error? `is_something_variant`?

Reply:
From the [mustache spec](https://mustache.github.io/mustache.5.html): 

> By default a variable "miss" returns an empty string. This can usually be configured in your Mustache library.

This works along with [this part of the spec](https://github.com/mustache/spec/blob/master/specs/sections.yml#L176-L180):

> Failed context lookups should be considered falsey.